deploy/cluster-monitoring-config-non-uwm/cluster-monitoring-config.yaml

```diff
@@ -70,13 +70,17 @@ data:
         - effect: NoSchedule
           key: node-role.kubernetes.io/infra
           operator: Exists
-    # https://access.redhat.com/solutions/5685771
-    # https://bugzilla.redhat.com/show_bug.cgi?id=1906496
+    openshiftStateMetrics:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+      tolerations:
+        - effect: NoSchedule
+          key: node-role.kubernetes.io/infra
+          operator: Exists
     thanosQuerier:
-      resources:
-        limits:
-          cpu: 500m
-          memory: 2048Mi
-        requests:
-          cpu: 5m
-          memory: 125Mi
```

Comment:
This should not be removed as this configuration is applied only to 4.5 per the `config.yaml`.  So, this applies only to impacted clusters.